app/controllers/logs_controller.rb

```diff
@@ -28,6 +28,10 @@ def create
     end
   end
 
+  def codes_only_export?(params)
```

Comment:
```suggestion
  def codes_only_export?
```
i think you already have access to this and therefore don't need to pass it on

Reply:
veyr good. Always nice to know where the bits of rails magic pop up regarding scope